frontend/src/libs/components/button/button.tsx

```diff
@@ -1,10 +1,15 @@
+import styles from './styles.module.scss';
+
 type Properties = {
   label: string;
   type?: 'button' | 'submit';
+  style?: 'submit';
```

Comment:
we will add it, when it will be needed, use current styles as default for now
```suggestion
```

Reply:
done